src/common/home/home.css

```diff
@@ -31,7 +25,7 @@
 
 .app-home-body .app-home-body-content {
   margin-bottom: 4rem;
-  z-index: 10;
+  z-index: 5;
```

Comment:
@yash91989201 , Just to confirm, this is the only change that you want to make right?

Reply:
Yes, the index value was only causing the content overlap

Reply:
Can you fix the formatting as the entire CSS file content is getting changed?

Reply:
Could you please suggest me how to do it?

Reply:
> Can you fix the formatting as the entire CSS file content is getting changed?

How is the file content getting changed??

Reply:
@yash91989201 , You can disable the formatting, so it will not format the CSS content.
 You can use this extension https://marketplace.visualstudio.com/items?itemName=tombonnike.vscode-status-bar-format-toggle.